_install_guide/logging.md

```diff
@@ -133,3 +134,66 @@ docker inspect -f '{{.HostConfig.LogConfig}}' clouddriver
 {% endraw %}```
 
 If the log config isn't what you expect then something is wrong in the Armory Spinnaker config, and if the config is what you expect the problem is likely in your distributed logging setup.
+
+## Monitoring Spinnaker With Datadog
+
+Spinnaker provides a monitoring container which exports metrics from the core sub-services. We'll need to add two additional containers to our docker-compose setup: `datadog` and `spinnaker-monitoring`.
+
+Add the following to `/opt/spinnaker/compose/docker-compose.override.yml`
```

Comment:
original message disappeared on new commit.
![](https://cl.ly/2h0L0f1L3T2z/Image%202018-04-03%20at%2017.48.16.png)

Reply:
the link: https://github.com/armory-io/armoryspinnaker/blob/master/src/spinnaker/bin/start#L110-L118